src/Talis/Persona/Client/Users.php

```diff
@@ -90,11 +101,19 @@ public function createUser($gupid, $profile, $token)
         }
         try
         {
-            $user = $this->personaPostUser($url, $query, $token);
-            return $user;
+            return $this->performRequest(
+                $url,
+                array(
+                    'method' => 'POST',
+                    'body' => json_encode($query),
+                    'bearerToken' => $token,
+                )
+            );
         } catch(\Exception $e)
         {
-            throw new \Exception('User not created');
+            $profileString = implode(',', $profile);
```

Comment:
could this variable name be improved.. does it need to be described as it's type?